native~/Shared/src/UnityAssetAccessor.cpp

```diff
@@ -39,13 +42,16 @@ class UnityAssetResponse : public IAssetResponse {
       : _statusCode(uint16_t(request.responseCode())),
         _contentType(),
         _data(std::move(handler.NativeImplementation().getData())) {
-    System::String contentTypeHeader =
-        request.GetResponseHeader(System::String("Content-Type"));
-    if (contentTypeHeader != nullptr) {
-      this->_contentType = contentTypeHeader.ToStlString();
-      this->_headers.emplace("Content-Type", this->_contentType);
+    System::Collections::Generic::Enumerator0 enumerator =
+        request.GetResponseHeaders().GetEnumerator();
+    while (enumerator.MoveNext()) {
+      std::string key = enumerator.Current().Key().ToStlString();
+      std::string value = enumerator.Current().Value().ToStlString();
+      if (value == "Content-Type") {
```

Comment:
Should this be `key == "Content-Type"`? Also HTTP headers are case-insensitive, so unless Unity is normalizing them we need to do a case-insensitive comparison. (The `GetResponseHeader` method we used before handles this automatically - maybe that easiest thing is to go back to retrieve the Content-Type)